test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -1615,6 +1615,35 @@ public void InstallCommand_AuthenticatedV3WithCredentialSavedInConfig()
             }
         }
 
+        /// <summary>
+        /// Unit test that proves non-zero exit code functionality when the number of
+        /// arguments for install command are not appropiate
+        /// </summary>
+        [Fact]
+        public void InstallCommand_WrongArguments()
```

Comment:
Consider using a similar name to similar tests: `InstallCommand_Failure_InvalidArguments_HelpMessage`